tests/benchmarks/test_startup.py

```diff
@@ -0,0 +1,6 @@
+from tests.shell import Command
+
+
+def test_startup(dvc_cmd: Command, benchmark):
+    result = benchmark(dvc_cmd.args("--help"))
+    assert result.returncode == 0
```

Comment:
We have `test_help` in `dvc-bench`. It should be under `cli/commands`, since we need an ablility to test any binary with this, no matter if it is installed with pip or deb/rom/etc.